rust/README.md

```diff
@@ -70,15 +70,11 @@ Alternatively, you can replace `cargo run` with `bazel run //rust:selenium-manag
-
+| **Milestone**                           | **Description**                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
+|-----------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| M1: Driver management                   | <ul><li>Beta version of the Selenium Manager.</li> <li>Focused on driver management for Chrome, Firefox, and Edge.</li> <li>Selenium Manager compiled for Windows, Linux, and macOS (in GH Actions).</li> <li>Available in Selenium binding languages (Java, JavaScript, Python, Ruby, and C#).</li> <li>Used as a fallback for language bindings, when the driver is not found.</li> <li>Selenium Manager binaries bundled within the binding languages.</li></ul> |
+| M2: Driver management for IEDriver      | <ul><li>Include driver support for IExplorer.</li> <li>Consider msedgedriver in IE Mode.</li></ul>                                                                                                                                                                                                                                                                                                                                                                  |
+| M3: Rich configuration                  | <ul><li>Proxy support in Selenium Manager.</li> <li>Extra configuration capabilities from binding languages to Selenium Manager (e.g., force to use a given driver version, etc.).</li></ul>                                                                                                                                                                                                                                                                        |
```

Comment:
Add ability to specify browser location (Se Mgr will check that location for version and grab the right driver for it).
Do we support beta/dev/canary right now?

Reply:
Not now, but it should be easy to implement. That can be part of the extra configuration milestone.